etc/kayobe/environments/ci-multinode/inventory/group_vars/seed/network-interfaces

```diff
@@ -10,6 +10,10 @@ external_interface: "{{ vxlan_interfaces[0].device }}.{{ external_vlan }}"
 
 public_interface: "{{ vxlan_interfaces[0].device }}.{{ public_vlan }}"
 
+# Only add the storage interface if we are using manila-cephfs.
+storage_interface_template: "{{ vxlan_interfaces[0].device }}.{{ storage_vlan }}"
+storage_interface: "{{ storage_interface_template if (kolla_enable_manila and kolla_enable_manila_backend_cephfs_native) else omit }}"
```

Comment:
This can be defined unconditionally. You will also need to add `storage` to `seed_extra_network_interfaces`, which could be done conditionally.